tests/mock_modules/testutils/fixtures_data.py

```diff
@@ -258,6 +258,19 @@ def get_summary(secure_arm=False, partition_ids=None,
                         'zone_type': 102,
                         'partition_id': 1,
                     },
+{
+                        'id': '002-0080',
+                        'type': 'Temperature',
+                        'name': 'My Temperature Sensor',
+                        'group': 'temperature',
```

Comment:
```suggestion
                        'group': 'Temperature',
```

As you mentioned in your comment (https://github.com/XaF/qolsysgw/pull/100#issuecomment-1446544139), the data here should be exactly what you see in your logs, except for the `zone_id`, `id` and `name` of course!